src/game/option_passport.c

```diff
@@ -130,6 +141,8 @@ static void Option_PassportShutdownText(void)
     Option_PassportFreeLevelStrings();
     m_PassportMode = PASSPORT_MODE_FLIP;
     m_IsTextInit = false;
+
+    Memory_FreePointer(&m_SavegameStrings);
```

Comment:
Need to also deallocate the individual strings.

```c
for (int i = 0; i < g_Config.maximum_save_slots; i++) {
    Memory_FreePointer(&m_SavegameStrings[i]);
}
```

This comes before deallocating the entire array.

Reply:
Should this not be `Memory_FreePointer(m_SavegameStrings[i]);`? Maybe I'm missing something but keeping the `&` crashes (because `m_SavegameStrings` already stores pointers, right?) and removing seems to fix the issue.